tasks/centos.yml

```diff
@@ -39,3 +39,16 @@
   tags:
     - zookeeper
     - services
+
+- name: "Set logging level"
+  lineinfile: >
+    dest=/etc/zookeeper/conf/log4j.properties
+    regexp='{{ item.regex }}'
+    line='{{ item.line }}'
+  with_items:
+    - { regex: "^zookeeper.root.logger=", line: "zookeeper.root.logger={{ zookeeper_log_level }}, CONSOLE" }
+    - { regex: "^zookeeper.console.threshold=", line: "zookeeper.console.threshold={{ zookeeper_log_level }}" }
+  when: ansible_distribution_major_version|int >= 7
```

Comment:
If these options are defined here, maybe they should be removed from `/etc/zookeeper/conf/zoo.cfg`.